docs/fundamentals/executor/hub/use-hub-executor.md

```diff
@@ -153,3 +153,11 @@ jina hub pull jinahub://<UUID>[:<SECRET>][/<TAG>]
 
 
 The source code of the Executor will be stored at `~/.jina/hub-packages`.
+
+### list the local location of executors
+
+```bash
+jina hub list
+```
+
+Show the local location of Jina executors.
```

Comment:
This is just rewording of the title. We need something better here, or just remove it altogether

Reply:
I will remove it and add the command line video.